### PR TITLE
chore: Logging for s3-related operations

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: packages/core/src/ceramic.ts [hunk 1]
```diff
@@ -354,7 +354,7 @@ class Ceramic implements CeramicApi {
     const ipfsTopology = new IpfsTopology(ipfs, networkOptions.name, logger)
     const pinStoreFactory = new PinStoreFactory(ipfs, pinStoreOptions)
     const repository = new Repository(streamCacheLimit, concurrentRequestsLimit, logger)
-    const dispatcher = new Dispatcher(ipfs, networkOptions.pubsubTopic, repository, logger, pubsubLogger)
+    const dispatcher = new Dispatcher(ipfs, networkOptions.pubsubTopic, repository, logger, pubsubLogger, loggerProvider.makeServiceLogger('dispatcher'))
```
Comment (stbrody) [inline packages/core/src/ceramic.ts:357]: I wonder if we should call this logger `ipfs`?  With the plan to one day pull out the functionality for loading/storing from IPFS out of the Dispatcher into its own class that's explicitly just for interacting with the IPFS dag.

**Review by v-stickykeys — APPROVED** [review]
👍

**Comment by stbrody** [comment]
@ukstv what's the status of this PR?

**Comment by ukstv** [comment]
No longer relevant.

**Pull request closed without merging**